purchase_order_report_selection/models/purchase_order.py

```diff
@@ -0,0 +1,56 @@
+from odoo import api, fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    report_field_ids = fields.One2many("purchase.report.field", "purchase_order_id")
+
+    @api.model
+    def create(self, value):
+        result = super(PurchaseOrder, self).create(value)
+        result.add_report_fields()
+        return result
+
+    def default_report_po_fields(self):
```

Comment:
The function doesn't have docstring and function must have protected type 
(function name started with `_ `. Example: `_default_report_po_fields`).

Reply:
Well, I will definitely add